server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMailboxesMethodStepdefs.java

```diff
@@ -68,6 +68,11 @@ public void assertMailboxesNames(String mailboxName) throws Exception {
         assertThat(httpClient.jsonPath.<List<String>>read(ARGUMENTS + ".list[*].name")).contains(mailboxName);
     }
 
+    @Then("^the mailbox \"([^\"]*)\" should not be present$")
```

Comment:
should not exist ?

Reply:
I prefer should not be present, as it can exist from an other user point of view

Reply:
ok, let's say "the mailboxes contain X"

Reply:
ok done